api/src/org/apache/cloudstack/api/command/user/template/DeleteTemplateCmd.java

```diff
@@ -52,6 +52,9 @@
     @Parameter(name = ApiConstants.ZONE_ID, type = CommandType.UUID, entityType = ZoneResponse.class, description = "the ID of zone of the template")
     private Long zoneId;
 
+    @Parameter(name = ApiConstants.FORCED, type = CommandType.BOOLEAN, required = false, description = "Force delete a template.")
```

Comment:
Please add `since` to this annotation to indicate that this parameter is available in 4.9+.